core/src/main/php/scriptlet/HttpScriptlet.class.php

```diff
@@ -44,6 +44,11 @@
    * @test    xp://net.xp_framework.unittest.scriptlet.HttpScriptletProcessTest
    */
   class HttpScriptlet extends Object {
+
+    // Constants from \xp\scriptlet\Runner
+    const
+      XML         = 0x0001,
```

Comment:
I don't like the duplication of magic constants here, but otherwise, we'd have to add a dependency to the `xp.scriptlet.WebDebug` class...